instrumentation-genai/opentelemetry-instrumentation-vertexai/src/opentelemetry/instrumentation/vertexai/__init__.py

```diff
@@ -56,6 +57,23 @@
 from opentelemetry.trace import get_tracer
 
 
+def _client_classes():
+    # This import is very slow, do it lazily in case instrument() is not called
+
+    # pylint: disable=import-outside-toplevel
+    from google.cloud.aiplatform_v1.services.prediction_service import (
```

Comment:
unwrap supports the module as string too so no need to import directly here if you don't want to